eox_core/settings/common.py

```diff
@@ -19,3 +21,8 @@ def plugin_settings(settings):
     """
     settings.EOX_CORE_USERS_BACKEND = "eox_core.edxapp_wrapper.backends.users_h_v1"
     settings.EOX_CORE_ENROLLMENT_BACKEND = "eox_core.edxapp_wrapper.backends.enrollment_h_v1"
+    settings.EOX_CORE_CERTIFICATES_BACKEND = "eox_core.edxapp_wrapper.backends.certificates_h_v1"
+    settings.EOX_CORE_COURSEWARE_BACKEND = "eox_core.edxapp_wrapper.backends.courseware_h_v1"
+    settings.EOX_CORE_GRADES_BACKEND = "eox_core.edxapp_wrapper.backends.grades_h_v1"
+    settings.DATA_API_DEF_PAGE_SIZE = 10
```

Comment:
this should be overridable via the json file

Reply:
Done @felipemontoya (aws.py file added)